persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -331,7 +331,7 @@ ORDER BY message.date ASC;
 
 getUnreadMessages:
 SELECT * FROM MessagePreview AS message
-WHERE isUnread
+WHERE isUnread AND isSelfMessage = 0
```

Comment:
will it be better if we do not mark self messages as unread when inserting?

Reply:
We don't have that value while inserting. We are comparing message date with last conversation read date